README.md

```diff
@@ -101,61 +108,57 @@ class Example extends React.Component {
+| value | array | | [] | List of images |
+| onChange | function | (imageList, addUpdateIndex) => void | | Called when add, update or delete action is called |
+| dataURLKey | string | | dataURL | Customized field name that base64 of selected image is assigned to |
+| multiple | boolean | | false | Set `true` for multiple chooses |
+| maxNumber | number | | 1000 | Number of images user can select if mode = `multiple` |
+| onError | function | (errors, files) => void | | Called when it has errors |
+| acceptType | array | ['jpg', 'gif', 'png'] | [] | The file extension(s) to upload |
+| maxFileSize | number | | | Max image size (Byte) and it is used in validation |
+| resolutionType | string | 'absolute' \| 'less' \| 'more' \| 'ratio' | | Using for image validation with provided width & height |
+| resolutionWidth | number | > 0 | | |
+| resolutionHeight | number | > 0 | | |
+
+### Note
+
+**resolutionType**
+
+| value             | description |
+| :---------------- | :---------- |
+| absolute | resolutionWidth === resolutionHeight |
+| ratio | (resolutionWidth / resolutionHeight) === (image width / image height) |
+| less | image's width < resolutionWidth && image's height < resolutionHeight |
+| more | image's width > resolutionWidth && image's height > resolutionHeight |
 
 ## Exported options
 
-| parameter        | type                                             | description                                           |
-| ---------------- | ------------------------------------------------ | ----------------------------------------------------- |
-| imageList        | array                                            | List of images for render\.                           |
-| onImageUpload    | function                                         | Call for upload new image\(s\)                        |
-| onImageRemoveAll | function                                         | Call for remove all image\(s\)                        |
-| onImageUpdate    | \(updateIndex: number\) => void                  | Call for update at updateIndex\.                      |
-| onImageRemove    | \(deleteIndex: number \| Array<number>\) => void | Call for remove one/list image\.                      |
-| errors           | object                                           | Export list of validation                             |
-| dragProps        | object                                           | Native element props for drag and drop feature        |
-| isDragging       | boolean                                          | "true" if image\(s\) is dragged into drag and space\. |
-
-## License
-
-MIT © [](https://github.com/)
+| parameter | type | description |
+| :---------------- | :------------------------------------------------ | :----------------------------------------------------- |
+| imageList | array | List of images to render. |
+| onImageUpload | function | Called when uploading new images |
```

Comment:
Is it clear for: if you want to open select file dialog, please trigger it?

Reply:
what do u suggest?